pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -12,6 +12,7 @@
 , proprietaryCodecs ? false, vivaldi-ffmpeg-codecs ? null
 , enableWidevine ? false, vivaldi-widevine ? null
 , commandLineArgs ? ""
+, pulseSupport ? false, libpulseaudio ? null
```

Comment:
Why did you not bind pulseSupport to `config.pulseaudio or stdenv.linux` as is done in chromium?

Reply:
Using native pulseaudio support is preferred, when possible. It won't hurt people with an alsa setup.

Reply:
i've updated the branch, would you like to test it as well?